src/AWSS3.jl

```diff
@@ -385,9 +459,18 @@ end
 s3_copy(a...; b...) = s3_copy(global_aws_config(), a...; b...)
 
 """
-    s3_create_bucket([:AbstractAWSConfig], bucket; kwargs...)
+    s3_create_bucket([::AbstractAWSConfig], bucket; kwargs...)
 
-[PUT Bucket](http://docs.aws.amazon.com/AmazonS3/latest/API/RESTBucketPUT.html)
+Creates a new S3 bucket with the globally unique `bucket` name. The bucket will be created
+AWS region associated with the `AbstractAWSConfig` (defaults to "us-east-1").
+
+# API Calls
+
+- [`CreateBucket`](https://docs.aws.amazon.com/AmazonS3/latest/API/API_CreateBucket.html)
+
+# Permissions
+
+- [`s3:DeleteBucket`](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazons3.html#amazons3-CreateBucket)
```

Comment:
I think you meant this?

```suggestion
- [`s3:CreateBucket`](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazons3.html#amazons3-CreateBucket)
```